SQL/paradise_schema.sql

```diff
@@ -604,3 +616,17 @@ CREATE TABLE `changelog` (
 	`cl_entry` TEXT NOT NULL,
 	PRIMARY KEY (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+--
+-- Table structure for table `ip2group`
+--
+DROP TABLE IF EXISTS `ip2group`;
+CREATE TABLE `ip2group`
+(
```

Comment:
```suggestion
CREATE TABLE `ip2group` (
```
Keep standardisation between creations